extensions/ql-vscode/src/qlpack-generator.ts

```diff
@@ -44,8 +52,19 @@ export class QlPackGenerator {
     await this.createCodeqlPackLockYaml();
   }
 
+  private async createWorkspaceFolder() {
```

Comment:
At the moment this is going to run behind a feature flag in the codespace template workspace: https://github.com/github/codespaces-codeql/pull/5

This is a multi-root workspace. 

We have planned work for making this behaviour work with the extension in the same way as the codespace. So we'll need to make sure this would work with a single root workspace. But for now, this PR is only aiming to work with the multi-root workspace.